recipes/flann/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    skip: True # [win]
+
+requirements:
+    build:
+        - gcc # [osx]
+        - hdf5
+        - cmake
+    run:
+        - libgcc # [osx]
+        - hdf5
+
+about:
+    home: http://www.cs.ubc.ca/research/flann/
+    license: BSD 3-Clause
+    license_file: COPYING
+    summary: FLANN: Fast Library for Approximate Nearest Neighbors
```

Comment:
Probably should quote this as it might confuse the yaml parser. That or get rid of the `:`. The latter seems a little better as we already know its FLANN, but both would work.